ait/core/server/server.py

```diff
@@ -117,15 +119,18 @@ def _load_streams(self):
                 common_err_msg.format(stream_type) + specific_err_msg[stream_type]
             )
             streams = ait.config.get(f"server.{stream_type}-streams")
-
             if streams is None:
                 log.warn(err_msgs[stream_type])
             else:
                 for index, s in enumerate(streams):
                     try:
                         if stream_type == "inbound":
                             strm = self._create_inbound_stream(s["stream"])
-                            if type(strm) == PortInputStream:
+                            if (
```

Comment:
Could this check be made into a static function in streams.py, something like `is_server_stream(strm)` ?